src/confluent_kafka/admin/__init__.py

```diff
@@ -976,12 +977,13 @@ def describe_user_scram_credentials(self, users, **kwargs):
         :raises ValueError: Invalid input value.
         """
         AdminClient._check_describe_user_scram_credentials_request(users)
-
         f, futmap = AdminClient._make_futures_v2(users, None,
                                                  AdminClient._make_user_scram_credentials_result)
 
-        super(AdminClient, self).describe_user_scram_credentials(users, f, **kwargs)
+        if len(users) == 0:
+            futmap["__all__"] = AdminClient._create_future()
```

Comment:
Shouldn't this be part of the above `_make_future` functions specific to this API?

Reply:
Actually no it is generic function and not specific to api otherwise we would have to change the signature of the api. As this is the only case for describe_user_scram api.